src/rust/include/rust/orchard/wallet.h

```diff
@@ -76,16 +74,60 @@ bool orchard_wallet_rewind(
         );
 
 /**
- * Searches the provided bundle for notes that are visible to the specified
- * wallet's incoming viewing keys, and adds those notes to the wallet.
+ * A C struct used to transfer action_idx/IVK pairs back from Rust across the FFI
+ * boundary. This must have the same in-memory representation as the `NoteMetadata` type
```

Comment:
This is now called `FFINoteMetadata` right? There are a few other places in this PR where the Rust struct is referred to as `NoteMetadata` still despite the name change.